### DISCUSSION TITLE
ZigStar LAN Gateway and ZigStar Stick v4 open source CC2652P based Zigbee adapters

### DISCUSSION THREAD

**Discussion by Hedda** [zigpy/zigpy#818] [General]
FYI, ZigStar LAN Gateway is a new C2652P based open source hardware Zigbee bridge by Radu mercenaruss

https://github.com/mercenaruss/zigstar_gateways

https://zig-star.com/projects/zigbee-gw-lan/

https://www.tindie.com/products/zigstar/zigstar-lan-gateway/

![image](https://user-images.githubusercontent.com/6320001/134335692-6f31da49-fa44-4910-ba3d-2ee47b486257.png)

mercenaruss also makes a ZigStar Stick v4 USB dongle with both SMA and uFL connectors for an external antenna as open-source hardware

https://github.com/mercenaruss/zigbee-stick-v4

https://mercenaruss.github.io/zigbee-stick-v4/

https://zig-star.com/projects/zigbee-stick-v4/

https://www.tindie.com/products/zigstar/zigstar-stick-v4/

![image](https://user-images.githubusercontent.com/6320001/134335654-ff469491-e126-4af1-9d13-9e37bea75caa.png)

**Comment by mercenaruss** [comment]
Thanks @Hedda for pointing this out, USB stick was available for opensource in November 2020,but didn't make a big thing about him.
Is very popular in Russian ZigBee Community.

Now i will concentrate on new projects related to ZigBee.

**Comment by mercenaruss** [comment]
User [xyzroe](https://github.com/xyzroe) come up with idea to build a customisable firmware for our LAN Coordinator.<br>
Firmware is based on  [ZiGate-Ethernet](https://github.com/fairecasoimeme/ZiGate-Ethernet).<br>
And was completely rewriten to match our PCB and required functions.

![LAN Coordinator Web Interface](https://user-images.githubusercontent.com/5508130/136787675-8aa340cc-4a0c-4d9f-be6b-c597c4524bf2.png)


In order to update your LAN Coordinator just upload ZigStarGW_vx.x.x.bin using web interface.<br>
For USB upgrade use:  ZigStarGW_vx.x.x.full.bin.

https://github.com/xyzroe/ZigStarGW-FW

**Reply by Hedda** [reply]
> User [xyzroe](https://github.com/xyzroe) come up with idea to build a customisable firmware for our LAN Coordinator.
> 
> Firmware is based on [ZiGate-Ethernet](https://github.com/fairecasoimeme/ZiGate-Ethernet).
> 
> And was completely rewriten to match our PCB and required functions.

Nice! As mentioned in the Home Assistant forum would suggest that you try to get Zeroconf working with ZHA integration if possible:

https://community.home-assistant.io/t/zha-automatic-network-discovery-of-zigbee-coordinator-bridges-gateways-ethernet-wifi-network-adapters-that-support-zeroconf/293300

Since you based your new firmware on ZiGate-Ethernet you might also be interested in this same request posted to fairecasoimeme:

https://github.com/fairecasoimeme/ZiGate-Ethernet/issues/7
